src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -156,6 +176,7 @@ export default {
 				};
 			}
 			this.fetchFilterData(this.flssLoanSearch);
+			this.loans = new Array(6).fill({ id: 0 });
```

Comment:
Is this to reset the loading state? Why was this not needed before?

Reply:
Right, having a similiar behavior in combo page for a better UX.
Not sure why we didn't consider before.

Reply:
Cool, sounds good.